load.php

```diff
@@ -24,6 +24,12 @@
 			'sites' => [],
 			'event-tracking' => false,
 		],
+		'integrations' => [
+			'segment' => [
+				'enabled' => true,
+				'api_key' => '',
+			],
+		],
```

Comment:
Wondering if the `enabled` flag is necessary here or if we just check the `api_key`. I think for consistency it's hyphens rather than underscores too, might need to look at some normalisation for that in future.

Reply:
Was accounting for the possibility of deep inheritance for environments' config. Using the API key as a status of activation seems like a workaround.